<launch>
    <param name="use_sim_time" value="false" />
    <param name="map_load_file" type="string" value="$(find cube_localization)/maps/GlobalMap.pcd"/>
    <param name="traj_load_file" type="string" value="$(find cube_localization)/maps/trajectory.pcd"/>

    <rosparam command="load" file="$(find cube_localization)/config/localization.yaml" />

    <!-- <node pkg="tf" type="static_transform_publisher" name="front_laser_broadcaster" args="0.65 0.36 1.48 0.0 0 3.14159 base_link front_livox 100" /> -->

    <node name="cube_localization" pkg="cube_localization" type="cube_localization" output="screen">
    </node>
    <include file="$(find fast_lio)/launch/mapping_mid360.launch">
    </include>


</launch>